is-it-the-first/index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html>
+<!--
+Oi, there's nothing to see here!
+Go on, off ya feck, ya nosey bastard!
+-->
+<head>
+	<link rel="stylesheet" href="../styles.css" />
+	<link rel="apple-touch-icon" sizes="180x180" href="favicon/apple-touch-icon.png">
+	<link rel="icon" type="image/png" sizes="32x32" href="favicon/favicon-32x32.png">
+	<link rel="icon" type="image/png" sizes="16x16" href="favicon/favicon-16x16.png">
+	<link rel="icon" type="image/x-icon" href="favicon/favicon.ico">
+	<meta property="og:type" content="website"/>
+	<meta property="og:url" content="https://nugigi.site/is-it-friday"/>
+	<meta property="og:title" content="Is it the first of the month?"/>
+	<meta property="og:description" content="Maybe. Click through to double check!"/>
+	<meta property="og:image" content="https://nugigi.site/SEAL.png"/>
+</head>
+<body class="nft">
+	<span class="sealContainer"><a href="../" title="Back to Intranet Home Page"><img class="seal" src="../SEAL.png" alt="A brown and dark red seal with NeRdfiGhTerIa INc. branded on it in bright green."/></a></span>
+	<div>
+	<h1>Is it a new month?</h1>
+	<span id="tada">
+    <p>Happy new month!</p>
+    <p>
+      <img src="clapping-mr-blobby.gif" alt="Mr Blobby clapping"/>
+      <img src="200.gif" alt="Elmo with his hands outstretched in front of a fire"/>
+      <img src="goose-dance.gif" alt="A goose dancing"/>
+    </p>
+  </span>
+  <span id="anyday">
```

Comment:
Close the <span> tag or it will also hide everything else below this (including the footer etc)